src/index.js

```diff
@@ -0,0 +1,163 @@
+import './style.css';
+import Item from './Item';
+import Storage from './Storage';
+
+window.storage = 'default';
+if (!localStorage.getItem('projects')) {
+  localStorage.setItem('projects', '');
+}
+
+function render() {
+  const body = document.getElementById('body');
+  body.innerHTML = '';
+  Item.toArray().forEach((item) => {
+    const tr = item.render();
+    const td = document.createElement('td');
+
+    const remove = document.createElement('a');
+    remove.addEventListener('click', (e) => {
+      e.preventDefault();
+      item.destroy();
+      render();
+    });
+    remove.innerHTML = 'Remove';
+
+    const toggle = document.createElement('a');
+    toggle.addEventListener('click', (e) => {
+      e.preventDefault();
+      item.toggle();
+      item.save();
+      render();
+    });
+    toggle.innerHTML = 'Toggle';
+
+    const edit = document.createElement('a');
+    edit.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      const form = document.querySelector('.form');
+      form.elements.id.value = item.id;
+      form.elements.title.value = item.title;
+      [form.elements.date.value] = item.date.toISOString().split('T');
+      form.elements.priority.value = item.priority;
+      form.elements.done.value = item.done;
+    });
+    edit.innerHTML = 'Edit';
+
+    td.append(remove);
+    td.appendChild(document.createTextNode(' | '));
+    td.append(toggle);
+    td.appendChild(document.createTextNode(' | '));
+    td.append(edit);
+
+    tr.append(td);
+    body.append(tr);
+  });
+}
+
+function addItemToLibrary(data) {
+  new Item(data).save();
+}
+
+const switchProject = (e) => {
+  e.preventDefault();
+  const { name } = e.target.dataset;
+
+  if (Storage.has(name) || name === 'default') {
+    document.querySelector('.current-project').innerHTML = `(${name})`;
+    window.storage = name;
+  }
+
+  render();
+};
+
+const deleteProject = (e) => {
+  e.preventDefault();
+
+  const { name } = e.target.dataset;
+
+  if (name !== 'default') {
+    document.querySelector('.current-project').innerHTML = '(Default)';
+    window.storage = 'default';
+    Storage.remove(name);
+    const projectsString = localStorage.getItem('projects') || '';
+    const projectsArray = projectsString.split(',').filter(i => i);
+    if (projectsArray.indexOf(name) > -1) {
+      delete projectsArray[projectsArray.indexOf(name)];
+    }
+    localStorage.setItem('projects', projectsArray.join(','));
+    e.target.remove();
+  }
+
+  render();
+};
+
+function renderProject(value) {
```

Comment:
* [ ] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/))
- let us redefine the method using` ES6` syntax